config/routes.rb

```diff
@@ -248,15 +237,30 @@
         patch :continue
         patch :reset
       end
+
+      scope module: :application_merits_task do
+        resource :statement_of_case, only: %i[show update destroy] do
+          get "/list", to: "statement_of_cases#list"
+        end
+        resource :opponent, only: %i[show update]
+        resource :date_client_told_incident, only: %i[show update]
+        resource :client_denial_of_allegation, only: %i[show update]
+        resource :client_offered_undertakings, only: %i[show update]
+        resources :involved_children, only: %i[new show update]
+        resource :has_other_involved_children, only: %i[show update]
+        resources :remove_involved_child, only: %i[show update]
+      end
     end
 
     resources :merits_task_list, only: [] do
-      resource :attempts_to_settle, only: %i[show update], controller: "proceeding_merits_task/attempts_to_settle"
-      resource :linked_children, only: %i[show update], controller: "proceeding_merits_task/linked_children"
-      resource :prohibited_steps, only: %i[show update], controller: "proceeding_merits_task/prohibited_steps"
-      resources :chances_of_success, only: %i[index create], controller: "proceeding_merits_task/chances_of_success"
-      resource :success_prospects, only: %i[show update], controller: "proceeding_merits_task/success_prospects"
-      resource :specific_issue, only: %i[show update], controller: "proceeding_merits_task/specific_issue"
+      scope module: :proceeding_merits_task do
+        resource :attempts_to_settle, only: %i[show update], controller: "attempts_to_settle"
```

Comment:
In an ideal world we wouldn't need to specify the controller, but we're too far down the road of pluralising `attempts_to_settle` in some places but not others to row back now.

This is the simplest way to move forwards without significant refactor, and keeps things as they were.

Reply:
Actually, are we too far down that road?

Has this part of the user journey been released to any users yet?

If it hasn't, I would advocate for pluralising the controller and updating the `step` to reflect that.

An alternative would be defining the inflection as uncountable and updating the table name etc, but that would be a bit more involved.

Interested to hear what others think.

cc @colinbruce because I feel like you will know what's been released to who and when in this area!

Reply:
The Attempts to settle question has been live for some months now, this is my least favourite part of rails 😄 
Trying to work out their grammar logic. attempts_to_settles is not and never will be acceptable! 😁